### DISCUSSION TITLE
How to check if dynarec enabled / disabled

### DISCUSSION THREAD

**Discussion by xDoge26** [ptitSeb/box86#705] [Q&A] [answered]
i want to disable dynarec temporary with "export BOX86_DYNAREC=0" but im not sure its disabled or not

**Comment by paulwratt** [comment]
for temporary disable, rather use:
```
BOX86_DYNAREC=0 bin-prg-here
```

**Comment by ptitSeb** [comment, marked as answer]
You can see if Dynarec is off only with `BOX86_LOG=1`
If you use `bOX86_LOG=1 BOX86_DYNAREC=0 ....` you will see `Dynarec if off` on the console. There is no specific message if Dynarec is on. This message will not be shown by default (`BOX86_LOG=0` by default)

(ony if Box86 was built with Dynarec. `box86 --version` will say `with Dynarec` if it's the case)
